01_fib/marksomething/main_test.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestFibStringPositive(t *testing.T) {
+
+	testCases := []struct {
+		desc     string
+		value    int
+		expected string
+	}{
+		{
+			desc:     "fib(0)",
+			value:    0,
+			expected: "0\n",
+		},
+		{
+			desc:     "fib(1)",
+			value:    1,
+			expected: "0\n1\n",
+		},
+		{
+			desc:     "fib(2)",
+			value:    2,
+			expected: "0\n1\n1\n",
+		},
+		{
+			desc:     "fib(10)",
+			value:    10,
+			expected: "0\n1\n1\n2\n3\n5\n8\n13\n21\n34\n55\n",
+		},
+	}
+	for _, tC := range testCases {
+		actual := fibString(tC.value)
+		if actual != tC.expected {
+			t.Errorf("\nActual:\n%v\nExpected:\n%v", actual, tC.expected)
+		}
+	}
+}
+func TestFibStringNegative(t *testing.T) {
+	testCases := []struct {
+		desc     string
+		value    int
+		expected string
+	}{
+		{
+			desc:     "fib(0)",
+			value:    0,
+			expected: "0\n",
+		},
+		{
+			desc:     "fib(-1)",
+			value:    -1,
+			expected: "0\n1\n",
+		},
+		{
+			desc:     "fib(-2)",
+			value:    -2,
+			expected: "0\n1\n-1\n",
+		},
+		{
+			desc:     "fib(-10)",
+			value:    -10,
+			expected: "0\n1\n-1\n2\n-3\n5\n-8\n13\n-21\n34\n-55\n",
+		},
+	}
+	for _, tC := range testCases {
+		actual := fibString(tC.value)
+		if actual != tC.expected {
```

Comment:
`testify/assert` => `assert.Equal(expected, actual)` if you wish

Reply:
much nicer :)